lib/app-router.js

```diff
@@ -84,6 +87,40 @@ module.exports = function (router) {
 
         break;
 
+      case 'apex':
+        let message = 'Apex Scripts run.';
+        async.eachSeries(param, function iteratee(apex, callback) {
```

Comment:
What affect do this have on the ajax calls? I had been running most things synchronously so that the output works properly.

Reply:
see next comment.